doc/documents/Examples_Tutorials/Examples_Tutorials.rst

```diff
@@ -0,0 +1,623 @@
+   |                           | .. code::                                                     |
+   |                           |                                                               |   
+   |                           |    layer {                                                    |
+   |                           |      name: "data"                                             |
+   |                           |      type: "Input"                                            |
+   | **ProtoText description** |      top: "data"                                              |
+   |                           |      input_param { shape: { dim: 1                            |
+   |                           |      dim: 3 dim: 32 dim: 32 } }                               |
+   |                           |    }                                                          |
+   |                           | ..                                                            |
+   +---------------------------+---------------------------------------------------------------+
+   |                           | .. code:: c++                                                 |
+   |                           |                                                               |   
+   |                           |    mli_status mli_krn_permute_fx8(                            |
+   |                           |        const mli_tensor * in,        // Input tensor          |
+   | **MLI Function**          |        const mli_permute_cfg * cfg,  // Permute configuration |
+   |                           |        mli_tensor * out              // Output tensor         |
+   |                           |      );                                                       |
+   |                           | ..                                                            |
+   +---------------------------+---------------------------------------------------------------+
+   |                           | .. code:: c++                                                 |
+   |                           |                                                               |   
+   |                           |    mli_permute_cfg permute_hwc2chw_cfg = {                    |
+   |                           |       .perm_dim =                                             |
+   |                           |           {FMAP_C_DIM_HWC, // 2                               |
+   |                           |            FMAP_H_DIM_HWC, // 0                               |
+   |                           |            FMAP_W_DIM_HWC} // 1                               |
+   |                           |    };                                                         |
+   |                           | ..                                                            |
+   +---------------------------+---------------------------------------------------------------+
+
+
```

Comment:
Was the "Next, we consider convolution and ReLU layers. " sentence omitted intentionally here?